VogtObserver4/ControlRoom.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver4
+{
+    class ControlRoom : RadiationMonitor
+    {
+        private double _warningThreshold;
+        private string _location;
+        private Object _state;
+        private double _average;
+
+        public ControlRoom(string location, double warningThreshold) : base(location)
+        {
+            _location = location;
+            _warningThreshold = warningThreshold;
+        }
+
+        public override string GenerateReport(double observation)
+        {
+            if (_warningThreshold < observation)
+            {
+                return Convert.ToString(observation);
+            }
+            else
+            {
+                return null;
```

Comment:
returning `null` in a `string` is not really meaningful. its not user friendly.

imagine seeing `null` in text as a non technical user of your program instead of something like.... `n/a` or something similar.

Reply:
if the intention is for this to be read by a human, of course

Reply:
guess you could be technically writing it out to a CSV for processing, in which case null might make sense. but you are printing it in a console os i think n/a would be more appropriate.